site/components/payment-streams/EditStream.js

```diff
@@ -150,12 +152,12 @@ const EditRate = function ({ stream }) {
 
 const EditFundingAddress = function ({ stream }) {
   const { active, account } = useWeb3React()
-  const { t } = useTranslation('payment-streams')
+  const t = useTranslations('PaymentStreams')
```

Comment:
for consistency, should we keep it as `payment-streams`?

```suggestion
  const t = useTranslations('payment-streams')
```

as none of the properties in the locale resource file use PascalCase

Reply:
With `next-translate` payment-streams had a separate file that was referenced with `useTranslation('payment-streams')`. 
With `next-intl` I did this in the same file but used a "namespace" as they suggested here (it's actually just a nested translation): https://next-intl-docs.vercel.app/docs/usage/messages#structuring-messages.
We could use `payment-streams` as the namespace, but that collides with the actual translation for `payment-streams` that is at the same level. 
But no problem, I can figure out another name for the namespace that uses kebab-case.

Reply:
oh, I did not notice the conflict of the properties. Yeah, maybe you can rename it so they all follow the same convention. Thanks!

Reply:
Changed `PaymentStreams` to `payment-streams-util` here: https://github.com/hemilabs/pure.finance/pull/1/commits/113feb3ac39ae44387c53a5400e9539d34c2ba4e